tableauserverclient/server/endpoint/auth_endpoint.py

```diff
@@ -73,3 +73,12 @@ def switch_site(self, site_item):
         self.parent_srv._set_auth(site_id, user_id, auth_token)
         logger.info('Signed into {0} as user with id {1}'.format(self.parent_srv.server_address, user_id))
         return Auth.contextmgr(self.sign_out)
+
+    @api(version="3.10")
+    def revoke_all_server_admin_tokens(self):
+        url = "{0}/{1}".format(self.baseurl, 'revokeAllServerAdminTokens')
+        if not self.parent_srv.is_signed_in():
+            return
+        self.post_request(url, '')
+        self.parent_srv._clear_auth()
```

Comment:
I don't think we need this line, since it'll end the user's current session. From my understanding, this endpoint will revoke all server admin PATs, but not log the calling user out.